src/components/dialogs/move-dialog.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { elementType } from '@gridsuite/commons-ui';
+import DirectorySelector from './directory-selector';
+import { useIntl } from 'react-intl';
+
+const MoveDialog = ({ open, onClose, items }) => {
+    const intl = useIntl();
+
+    //TODO : change nbElement
+    return (
+        <>
+            <DirectorySelector
+                types={elementType.DIRECTORY}
+                open={open}
+                onClose={onClose}
+                title={intl.formatMessage({ id: 'moveItemTitle' })}
+                validationButtonText={intl.formatMessage(
```

Comment:
can use <FormattedMessage id={...} /> instead of intl.formatMessage({id=... });
supress the need of hook useIntl() ;)